R/SurveyData.R

```diff
@@ -156,11 +156,6 @@ SurveyData <- R6::R6Class(
           call. = FALSE
         )
       }
-      if (ncol(data) != length(questions) &
-          length(questions) == sum(stats::complete.cases(data))) {
-        stop("Mismatch between number of data columns and questions.",
-             call. = FALSE)
-      }
```

Comment:
unrelated to this PR but I removed this check because I couldn't figure out why it's here. `sum(stats::complete.cases(data))` will give the number of _rows_ with complete data but that number isn't related to how many questions (columns) there are. So I don't think this check is needed. But I could easily be wrong here, in which case we can add this back in.